### DISCUSSION TITLE
Kali Linux on ISH Alpine

### DISCUSSION THREAD

**Discussion by JFadeHacks** [ish-app/ish#2455] [Q&A]
I'm new to git and whole Linux and im trying to learn i have IOS iPad Pro, and i have Kali 2.0 on ISH APP which is originally alpine.  My question is... Are the terminal codes the same as Kali on different device or is it a little different with Kali on Alpine ? 

Where can i find best tutorial to learn how to use Kali through ISH to use best tools Kali is mainly used for? 

Thank you for your help !

**Comment by jaclu** [comment]
Kali would be mostly pointless on iSH, since it does not have access to the networking layer
